### PR TITLE
Add set-kubelogin to database backup workflow

### PR DESCRIPTION
**Pull request by johnake**
### Context

Add set-kubelogin to database backup workflow, as this is required after adding rbac authentication

### Checklist

-   [ ] Attach to Trello card
-   [ ] Rebased master
-   [ ] Cleaned commit history
-   [ ] Tested by running locally

### CODE REVIEW

File: .github/workflows/backup-db.yml [hunk 1]
```diff
@@ -30,6 +30,10 @@ jobs:
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
+      - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
```
Comment (saliceti) [inline .github/workflows/backup-db.yml:33]: kubelogin is used by get-cluster-credentials so it must happen before it

**Comment by saliceti** [comment]
The confirmation must be skipped in the workflow. please use the same implementation as in the template: https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/Makefile#L39

File: .github/workflows/backup-db.yml (hunk 1 above)
Comment (johnake) [reply .github/workflows/backup-db.yml:33]: That's already above get-credentials? I just removed the duplicate before it

File: Makefile [hunk 2]
```diff
@@ -130,7 +130,7 @@ ci:	## Run in automation environment
 	$(eval DISABLE_PASSCODE=true)
 	$(eval AUTO_APPROVE=-auto-approve)
 	$(eval SP_AUTH=true)
-	$(eval CONFIRM_DEPLOY=true)
+	$(eval SKIP_CONFIRM=true)
```
Comment (saliceti) [inline Makefile:133]: This is only half implemented. Please check the template to see how SKIP_CONFIRM is used

File: Makefile [hunk 3]
```diff
@@ -128,9 +128,11 @@ set-azure-pd-subscription:
 
 ci:	## Run in automation environment
 	$(eval DISABLE_PASSCODE=true)
-	$(eval AUTO_APPROVE=-auto-approve)
 	$(eval SP_AUTH=true)
 	$(eval CONFIRM_DEPLOY=true)
+	$(eval AUTO_APPROVE=-auto-approve)
+	$(eval SKIP_AZURE_LOGIN=true)
```
Comment (saliceti) [inline Makefile:134]: This is not used, please remove

**Review by saliceti — APPROVED** [review]

**Pull request merged**